meta/main.yml

```diff
@@ -14,7 +19,7 @@ galaxy_info:
   # - GPLv3
   # - Apache
   # - CC-BY
-  license: license (GPLv2, CC-BY, etc)
+  license: MIT
 
   min_ansible_version: 1.2
```

Comment:
This version is worth bumping, with the comment that is should not be greater than the oldest supported Ansible version.